### PR TITLE
Format complete ISS experiment and expose in starfish.data

### PR DESCRIPTION
**Pull request by ambrosejcarr**
* update starfish.data with complete ISS experiment
* add coordinates to experiment
* update TileFetcher

Testing strategy: 
- Ran notebook with new data, got same results for `fov_000` in new experiment (is named `fov_001` in master)

### CODE REVIEW

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py [hunk 1]
```diff
@@ -9,30 +9,27 @@
...
 
+from starfish.core.util.argparse import FsExistsType
 from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
 from starfish.types import Axes, Coordinates, Number
-from starfish.util.argparse import FsExistsType
 
 
 class IssCroppedBreastTile(FetchedTile):
-    def __init__(self, file_path):
+    def __init__(self, file_path, coordinates):
```
Comment (ttung) [inline docs/source/_static/data_formatting_examples/format_iss_breast_data.py]:
```suggestion
    def __init__(self, file_path: str, coordinates: Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]):
```

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py [hunk 2]
```diff
@@ -59,18 +58,36 @@ def round_dict(self):
...
+        coordinates = {
+            Coordinates.X: (
+                self.coordinates_df.loc[fov, "x_min"],
+                self.coordinates_df.loc[fov, "x_max"]
+            ),
+            Coordinates.Y: (
+                self.coordinates_df.loc[fov, "y_min"],
+                self.coordinates_df.loc[fov, "y_max"]
+            ),
+            Coordinates.Z: 0
```
Comment (ttung) [inline docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: Does this experiment have multiple Z layers?  If not, I think you can just omit this.

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py [hunk 3]
```diff
@@ -84,13 +101,23 @@ def get_tile(self, fov: int, r: int, ch: int, z: int) -> FetchedTile:
...
```
Comment (ttung) [inline docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: ditto.

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py [hunk 4]
```diff
@@ -122,12 +149,18 @@ def add_codebook(experiment_json_doc):
...
         },
-        postprocess_func=add_codebook,
-        default_shape={Axes.Y: 1044, Axes.X: 1390}
     )
 
 
 if __name__ == "__main__":
+    """
+    This TileFetcher should be run on data found at:
+    s3://spacetx.starfish.data/mignardi_breast_1/raw/
```
Comment (ttung) [inline docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: do we want to move this sucker?

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py [hunk 5]
```diff
@@ -122,12 +149,18 @@ def add_codebook(experiment_json_doc):
             'nuclei': ISSCroppedBreastAuxTileFetcher(input_dir, 'nuclei'),
             'dots': ISSCroppedBreastAuxTileFetcher(input_dir, 'dots'),
         },
-        postprocess_func=add_codebook,
-        default_shape={Axes.Y: 1044, Axes.X: 1390}
     )
 
 
 if __name__ == "__main__":
+    """
```
Comment (ttung) [inline docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: consider moving this to be the `description` argument passed to `argparse.ArgumentParser()`

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py (hunk 4 above)
Comment (ambrosejcarr) [reply docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: Yes, we should move this.

File: docs/source/_static/data_formatting_examples/format_iss_breast_data.py (hunk 5 above)
Comment (ambrosejcarr) [reply docs/source/_static/data_formatting_examples/format_iss_breast_data.py]: If we want these scripts to be user-facing and user-friendly there's a lot of work to be done across them; right now we have a mixture of click and argparse scripts. If we think it's worth cleaning these up I'd suggest we do them all in a separate PR.

**Pull request merged**